### DISCUSSION TITLE
ABILITARE S1 e S2 su SONOFF MINI TASMOTA

### DISCUSSION THREAD

**Discussion by valerio54** [indomus/forum#460] [Hardware & Firmware]
Salve.
ho caricato il FW tasmota.bin ver. 9.5.0 su un mini sonoff , quando tento di abilitare s1 e s2 caricando e abilitando su "other parameters la stringa : 
{“NAME”:“Sonoff Mini”,“GPIO”:[17,0,0,0,9,0,0,0,21,56,0,0,255],“FLAG”:0,“BASE”:1} , il sonoff mi rimane sempre in "generico" come se non accettasse il parametro:"Sonoff Mini" .
A qualcuno è capitata la stessa cosa? , come si può risolvere?
Grazie

**Comment by valerio54** [comment]
sono riuscito dopo vari tentativi a caricare il parametro " Sonoff Mini" ma S1 e S2 ancora non funzionano. Qualcuno può dirmi se è possibile abilitarli tramite comandi in "console" ???

**Reply by snakuzzo** [reply]
Il per il sonoff mini ho usato questo template...e lo utilizzo con un interruttore

`{"NAME":"Sonoff Mini","GPIO":[32,0,0,0,160,0,0,0,224,320,0,0,1,0],"FLAG":0,"BASE":1}`

**Reply by valerio54** [reply]
si , ho provato anche io a caricare uguale al tuo , ma è sempre la stessa cosa S1 e S2 non funzionano

**Reply by snakuzzo** [reply]
scusa, ma qual è lo schema elettrico che hai usato ?

**Reply by essegi65** [reply]
Scusate ma nel sonoff mini siete riusciti a farci stare il firmware Tasmota.bin e non il Tasmota-lite.bin?
A me rende errore a fare l'aggiornamento con il firmware Tasmota completo.

**Reply by snakuzzo** [reply]
Si può mettere Tasmota completo...non ci guadagni praticamente niente rispetto alla lite sul sonoff mini, ma si può fare
Leggi [QUI](https://indomus.it/guide/riprogrammare-un-itead-sonoff-usando-la-modalita-diy-via-ota/) ... sezione **Scelta Firmware**

**Reply by valerio54** [reply]
Cosa intendi per schema elettrico? quello usato per caricare il FW sul sonoff mini???

**Reply by snakuzzo** [reply]
No...quello che hai usato per collegarlo all'impianto e all'interruttore/pulsante

**Reply by valerio54** [reply]
Ho solo alimentato il sonoff mini e messo in corto S1 e S2 , ma il relè non commuta . Ho altri sonoff mini con uguale FW e funzionano bene. L'unica possibilità è che questo sonoff si sia guastato , ma la cosa è strana , dato che fino a quando non l'ho aggiornato al nuovo FW funzionava. Ho anche provato a ricaricare le versione precedente del FW ma il risultato non cambia, ecco perchè chiedevo se c'era la possibilità di abilitare S1 e S2 tramite comandi in console

**Reply by essegi65** [reply]
Scusa cosa vuol dire "aver messo in corto S1 e S2"? i due poli S1 e S2 servono per avere la tipica funzione di interruttore o pulsante (senza fase) quindi se premi l'interruttore ad essi collegato il relè del sonoff mini si eccita e se ripremi il relè si stacca. Sicuro di aver fatto questo? Se non colleghi i 2 poli S1 e S2 il sonoff mini lo puoi comandare solo da remoto

**Reply by valerio54** [reply]
No, non funziona così, quando metti in corto s1 e S2 il relè eccita, quando togli il corto il relè cade, provaci e vedi che questo è il normale funzionamento
